src/redux/planCandidate.ts

```diff
@@ -40,6 +40,7 @@ export type PlanCandidateState = {
     savePlanFromCandidateRequestStatus: RequestStatus | null;
     updatePlacesOrderInPlanCandidateRequestStatus: RequestStatus | null;
     updateLikeAtPlaceInPlanCandidateRequestStatus: RequestStatus | null;
+    AutoReorderPlacesInPlanCandidateRequestStatus: RequestStatus | null;
```

Comment:
変数名は小文字スタートでお願いしますー
```suggestion
    autoReorderPlacesInPlanCandidateRequestStatus: RequestStatus | null;
```

[fyi] [JavaScript Style Guide # Naming Conventions](https://www.w3schools.com/js/js_conventions.asp#Naming%20Conventions:~:text=Try%20it%20Yourself%20%C2%BB-,Naming%20Conventions,-Always%20use%20the)

Reply:
完全に抜けてました👍